common/lib/xmodule/xmodule/templates/problem/checkboxes_response.yaml

```diff
@@ -3,21 +3,46 @@ metadata:
     display_name: Checkboxes
```

Comment:
Hi @nparlante : we're going to ask that we don't include any of these changes to any of hte problem yaml templates. While useful information, it's much more in-depth than we want the basic problem templates to be.

However, please don't revert the changes to these templates just yet. They provide a lot of useful context & examples to the docs team that will be helpful as they write documentation. Please revert these changes just prior to merging.

Speaking of merging: I'm going to work with the final reviewers of this PR (technical & documentation) to figure out when they can schedule in review. I'll reach out to you out-of-band with a rough estimate of when that will be, so that you can be on point to promptly respond to feedback and we can do one final, concerted push to get this in.